src/pages/Version.tsx

```diff
@@ -97,14 +89,8 @@ export const VersionPage: React.VFC = () => {
     },
   });
 
-  // This resets the pages states to force showing the loading state when the page is changed on navigation
-  useEffect(() => {
-    setIsLoadingData(true);
-    setRedirectURL(undefined);
-  }, [id]);
```

Comment:
I deleted this hook. Not sure if it's important but it didn't seem to do anything